src/Evolution/Systems/GrMhd/ValenciaDivClean/BoundaryCorrections/BoundaryCorrection.hpp

```diff
@@ -12,6 +12,7 @@
 /// Boundary corrections/numerical fluxes
 namespace grmhd::ValenciaDivClean::BoundaryCorrections {
```

Comment:
oh and this is doxygen version dependent on whether `BoundaryCorrections` is documented